src/standalone/Soil/boundary_conditions.jl

```diff
@@ -516,7 +519,8 @@ layer cell index.
 If `F_bc` can be approximated as independent of `Y_N`, the derivative
 is zero.
 """
-function ClimaLand.∂tendencyBC∂Y(
+# TODO do we even need to define this method? won't be called if not using MoistureStateBC at TopBoundary
```

Comment:
agreed - cant we fall back on https://github.com/CliMA/ClimaLand.jl/pull/542/files#diff-a450f17176137e18c75988ec46e63ededf61ce96a207f9d816a7777cf8432d2bL60?

Reply:
Also, we can fall back on the default and not use the `if haskey()` statement!